templates/job-filters.php

```diff
@@ -39,7 +39,7 @@
 		<?php if( apply_filters( 'job_manager_job_filters_show_remote_position', get_option('job_manager_enable_remote_position', true ), $atts ) ) : ?>
 			<div class="search_remote_position">
 				<input type="checkbox" class="input-checkbox" name="remote_position" id="remote_position" placeholder="<?php esc_attr_e( 'Location', 'wp-job-manager' ); ?>" value="1" <?php checked(! empty( $remote_position ) ); ?> />
-				<label for="remote_position" id="remote_position_label"><?php esc_html_e( 'Remote positions only', 'wp-job-manager' ); ?></label>
+				<label for="remote_position" id="remote_position_label"><?php esc_html_e( 'Remote listings only', 'wp-job-manager' ); ?></label>
```

Comment:
Although I can see a benefit in being consistent, I would prefer not to change a frontend facing label for the sake of it. I also think that positions is a better word here.

```suggestion
				<label for="remote_position" id="remote_position_label"><?php esc_html_e( 'Remote positions only', 'wp-job-manager' ); ?></label>
```